src/main/java/com/sap/oss/phosphor/fosstars/model/score/oss/ArtifactReleaseHistoryScore.java

```diff
@@ -0,0 +1,179 @@
+package com.sap.oss.phosphor.fosstars.model.score.oss;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.RELEASED_ARTIFACT_VERSIONS;
+import static java.time.temporal.ChronoUnit.DAYS;
+
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures;
+import com.sap.oss.phosphor.fosstars.model.score.FeatureBasedScore;
+import com.sap.oss.phosphor.fosstars.model.value.ArtifactVersion;
+import com.sap.oss.phosphor.fosstars.model.value.ArtifactVersions;
+import com.sap.oss.phosphor.fosstars.model.value.ScoreValue;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.IntSummaryStatistics;
+import java.util.Iterator;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+/**
+ * The scores assesses how well maintained an open source library is (in a specific version).
+ * <ul>
+ *   <li>{@link OssFeatures#RELEASED_ARTIFACT_VERSIONS}</li>
+ * </ul>
+ */
+public class ArtifactReleaseHistoryScore extends FeatureBasedScore {
```

Comment:
Looks like the methods for calculating stats in this class may be static. It would be good to add unit tests for them since they look quite complex.

Reply:
For the `ArtifactReleaseHistoryScore` I have an enhancement in mind for which I have created issue #479.
Hence, I now create only basic tests and extend tests as the test anyways have to be changed during #479.

Reply:
This issue doesn't say anything about improving tests. I assume you don't forget about it.

Reply:
Hope it is fine that I changed visibility of the static methods to package so that they are accessible in the tests.

Reply:
That is fine.